### PR TITLE
Docs: recreates setup for oss alerting

### PR DESCRIPTION
**Pull request by brendamuir**
creates new set up for OSS
adds set up for cloud (taken from legacy alerting docs)
for the moment they'll both display here with labels for either oss or cloud - and later will be moved to the website repo for cloud-only content

### CODE REVIEW

File: docs/sources/alerting/set-up/_index.md [hunk 1]
```diff
@@ -1,15 +1,61 @@
...
+1. Configure notification policies
+
+   - Check the default notification policy
+
+   - [Optional] Add additional nested policies
+
+   - [Optional] Add labels and label matchers to control alert routing
+
+1. [Optional] Integrate with [Grafana OnCall]
+   (/docs/oncall/latest/integrations/grafana-alerting)
```
Comment (gillesdemey) [inline docs/sources/alerting/set-up/_index.md:50]: This link doesn't seem to render for me, intentional?

**Review by gillesdemey — APPROVED** [review]
LGTM, minor nits only

File: docs/sources/alerting/set-up/_index.md [hunk 2]
```diff
@@ -1,15 +1,61 @@
...
-Alerting supports a plethora of configurations, from [configuring external Alertmanagers]({{< relref "./configure-alertmanager" >}}) to routing Grafana Managed Alerts outside of Grafana, to defining your alerting setup [as-code using Provisioning]({{< relref "./provision-alerting-resources" >}}).
+Configure the features and integrations that you need to create and manage your alerts.
 
-Setup Grafana Alerting high-availability mode by [following this guide]({{< relref "./configure-high-availability" >}}).
+**Note:**
 
-Connect your alerting setup to Grafana OnCall by [following this guide](/docs/oncall/latest/integrations/available-integrations/add-grafana-alerting/).
+These are set up instructions for Grafana Alerting Open Source.
+
+To set up Grafana Alerting for Cloud, see
```
[the remark this answers is not in the document]
Comment (brendamuir) [reply docs/sources/alerting/set-up/_index.md:20]: ah yes--great catch -- placeholder that i forgot about :)

File: docs/sources/alerting/set-up/_index.md (hunk 1 above)
Comment (brendamuir) [reply docs/sources/alerting/set-up/_index.md:50]: fixed

**Pull request merged**